### PR TITLE
Improve home page

### PR DESCRIPTION
**Pull request by MarcSkovMadsen**
I've tried to give the home page an update including

- Simplify some of the wording
- Add some of the missing but useful information that you need from an overview

You can *read* the `Github` rendered file [here](https://github.com/holoviz/panel/blob/c330da409d9fae5e09d5f8aaff6affac057c79d4/doc/index.md)

### CODE REVIEW

File: doc/index.md [hunk 1]
```diff
@@ -12,6 +12,35 @@ width: 125
 
 <h2 style="margin-top: 0.3em;">A high-level app and dashboarding solution for Python</h2>
 
+Panel is an [open-source and free](https://github.com/holoviz/panel/blob/main/LICENSE.txt) Python library that lets you create interactive tools, dashboards and data apps by connecting user-defined widgets to plots, images, tables, text and more.
```
Comment (MarcSkovMadsen) [inline doc/index.md:15]: I've mainly moved the apps down. I'm in doubt here. Currently I hope users don't click on the apps to check them out. They are simply not of a high enough quality to showcase Panel for real.

**Comment by MarcSkovMadsen** [comment]
I actually thought you (@philippjfr) had closed [my other homepage PR](https://github.com/holoviz/panel/pull/4216). Thought I saw that in my notifications a while back.

This one is simple to merge. Its in the informal style that some PyData packages has. The other one is more visual trying to sell Panel via visualizations and gifs.

It would be nice to know which style you prefer @philippjfr ??

**Comment by philippjfr** [comment]
Not sure yet, maybe neither but that doesn't mean your work is wasted. I've already included some of the changes you made elsewhere.

**Comment by MarcSkovMadsen** [comment]
Abandoned in favor of https://github.com/holoviz/panel/pull/4674

**Pull request closed without merging**